components/org.wso2.carbon.identity.api.server.oidc.scope.management/org.wso2.carbon.identity.api.server.oidc.scope.management.v1/src/gen/java/org/wso2/carbon/identity/api/server/oidc/scope/management/v1/factories/OidcApiServiceFactory.java

```diff
@@ -21,10 +21,9 @@
 
 public class OidcApiServiceFactory {
 
-   private final static OidcApiService service = new OidcApiServiceImpl();
+    private final static OidcApiService SERVICE = new OidcApiServiceImpl();
```

Comment:
Please check formatting

Reply:
I verified with wso2 code styles and formatted.